tools/docker-entrypoint.sh

```diff
@@ -2,18 +2,20 @@
 
 : "${EXPOSE_TCP:=false}"
 
+networkdatadir="${LIGHTNINGD_DATA}/${LIGHTNINGD_NETWORK}"
+
 if [ "$EXPOSE_TCP" == "true" ]; then
     set -m
     lightningd "$@" &
 
     echo "C-Lightning starting"
-    while read -r i; do if [ "$i" = "lightning-rpc" ]; then break; fi; done \
-    < <(inotifywait  -e create,open --format '%f' --quiet "$LIGHTNINGD_DATA" --monitor)
+    while read i; do if [ "$i" = "lightning-rpc" ]; then break; fi; done \
```

Comment:
Hmm, this seems to be different.  `-r` for `read` means "ignore backslashes on input", which is recommended.  `shellcheck` is also complaining about this as well.  Maybe should not be removed?

Reply:
Yes, it is my fault. fixed.